src/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require 'functions.php';
+
+echo produce_HTML_Head();
+$db = new PDO('mysql:host=db; dbname=cosmic_collector', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$cosmic_event_query = $db->prepare('SELECT * FROM `cosmic_events`;');
+$cosmic_event_query->execute();
+$cosmic_events = $cosmic_event_query->fetchAll();
```

Comment:
put this code at the top right under require, it should be executed before any html is echoed

Reply:
Done